muxers/mplex/src/codec.rs

```diff
@@ -28,7 +28,7 @@ use unsigned_varint::{codec, encode};
 // Arbitrary maximum size for a packet.
 // Since data is entirely buffered before being dispatched, we need a limit or remotes could just
 // send a 4 TB-long packet full of zeroes that we kill our process with an OOM error.
-const MAX_FRAME_SIZE: usize = 32 * 1024 * 1024;
+const MAX_FRAME_SIZE: usize = 1 << 20;
```

Comment:
For people like me who are not as much into bit twiddling, `1024 * 1024` would be easier to read.

Reply:
I was on the fence on that myself – had to calc it to verify it was the same – but it's what Go does so I thought I'd just copy it. But I'll change.